src/PrivateKey.ts

```diff
@@ -47,21 +47,22 @@ export class PrivateKey {
     }
 
     /** Retrieve the public key from a private key */
-    public getPublicKey(): PublicKey {
+    public privateToPublic(): PublicKey {
         const ellipticPrivateKey = this.toElliptic();
         return PublicKey.fromElliptic(ellipticPrivateKey, this.getType(), this.ec);
     }
 
-    /** Sign a message digest with private key */
-    public sign(digest: BNInput): Signature {
+    /** Sign a message or hashed message digest with private key */
+    public sign(data: BNInput, shouldHash: boolean = true, encoding: string = 'utf8'): Signature {
```

Comment:
`shouldHash` default should be false for backwards compatibility.

Reply:
`eosjs-ecc` `sign` hashes the data by default, hence `shouldhash` is set to true by default.

Reply:
This comment was only related to backwards compatibility if we released this as `21.x.0`